translations/strings.yaml

```diff
@@ -541,3 +541,19 @@ inAppAuth:
   termsOfServiceAndPrivacyDisclaimer:
     value: By continuing, you agree to the <a href="terms-of-service">Terms of Service</a> and the <a href="privacy-notice">Privacy Notice</a>.
     comment:  Do not remove or translate the markup '<a href="terms-of-service">', '<a href="privacy-notice">', '</a>'.
+
+#Accessibility
+accessibility:
+  currentIndexFocusedOfTotalItemsInGroup:
+    value: "%1 of %2"
+    comment:
+      - Used by screen readers to explain to users which item they are currently focused on, and how many items there are total in a group.
+      - For example, given a list with 15 items. If a user selects the 3rd item, we want the screen reader to read out "3 of 15".
+      - The %1 represents the index of the currently focused item by the screen reader.
+      - The %2 represents the total number of items in the group.
+  selectedAndItemName:
+    value: "Selected, %1"
```

Comment:
As mentioned, this is tricky, because you'd need to know what it references (city, country, something else completely, given this looks reusable). 

Could we add a noun, e.g. `Selected item: %1`, so that languages can safely match the gender?

Reply:
Hmmm, I suppose we could do that. 

Just to clarify and make sure it is actually necessary, the word "Selected" here is basically it's own sentence (and we can make the comma a period to reflect that if that makes more sense). If we look at the example in the video from the PR summary, if single-hop is currently selected, then the screen reader should read out "Selected. Single hop. Button. 1 of 2". If "Selected" is it's own sentence, then does the rest of the context still matter?

Sorry for all the back and forth, as you can tell, I am not well versed in other languages 😄

Reply:
It's not important if "selected" is contained in a stand-alone sentence.

Practical example in Italian:
- If referred to a button (`pulsante`), you need to decline adjective as male (selected = selezionat**o**).
- If referred to an image (`immagine`), you need to decline it as female (selected = selezionat**a**).

Most of the major languages need this, English is the exception.

You could use male, and hope that it doesn't create confusion. But having an actual noun removes any risk.

BTW, in your example, where does the translation for "Button" come from? From the OS?

Reply:
The only hesitation I have with adding "item" after "selected" is that in a future case (not in the example in this PR), we may have components where more than one item can be selected, and I am afraid that saying "selected item" may sound like only one item can be selected, and refers to the item as part of a group, whereas just "selected" ignores the idea that the item is a part of a group, and only focuses on whether the current item is selected or not. Does that make sense?

Also, you are correct, "button" comes from and is translated by the OS

Reply:
I have bad news: adjectives need to be declined for gender and number. So, that scenario with multiple selected items makes things worse, also considering we don't support plurals :-( 

If we need to support that, we need two strings: `Selected item: %1` and `Selected items: %1`

Reply:
Haha, I guess we can worry about that when the time comes. I have changed the string to "Selected item: %1". Let me know if there is anything else.

Thanks for all of the insight!